hypothesis-python/src/hypothesis/internal/entropy.py

```diff
@@ -90,15 +90,15 @@ def restore_all():
 
 
 @contextlib.contextmanager
-def deterministic_PRNG():
+def deterministic_PRNG(seed=0):
```

Comment:
Is this left over from an earlier revision of the PR?

It doesn't appear to be used by the main changes in `pytestplugin.py`.

Reply:
It's not actually used, but I thought it was a clearly better API.